hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestConnectionManager.java

```diff
@@ -301,4 +301,14 @@ public void testConfigureConnectionActiveRatio() throws IOException {
 
     tmpConnManager.close();
   }
+
+  @Test
+  public void testUnsupportedProtoExceptionMsg() throws IOException {
+    exceptionRule.expect(IllegalStateException.class);
+    exceptionRule.expectMessage(UnsupportedProto.class.getName());
```

Comment:
Can we put the full string here?
BTW, what was the old message exactly?

Reply:
java.lang.Class